Applications/ISIS.md

```diff
@@ -0,0 +1,65 @@
+## Introduction
+The ISIS software project is a collection of planetary image processing applications for data collected by a wide range of sensors.
+
+## History & Metrics
+Originating back in the 1970s, ISIS has a long history that is detailed [here](https://isis.astrogeology.usgs.gov/documents/IsisHistory/IsisHistory.html). Recently, the ISIS project has started transitioning to open source and is seeking greater community involvement.
+
+Currently, ISIS has 34 code contributors on Github. The majority of them are employees or past employees of the Astrogeology Science Center (ASC). In the last year, a small number of non-ASC personnel have contributed to the project.
+
+The ISIS software package has been downloaded from Anaconda cloud over a thousand times in the last year.
+
+## Scope
+The ISIS software project encompasses software used for photometric and photogramettric processing of planetary remote sensing data.
+
+The ISIS working group will have sole responsibility and discretion over the Planetary Software
+project in the following areas:
```

Comment:
So the object here is 'the Planetary Software project.'  Should that be 'the ISIS project?'